requirements-test.txt

```diff
@@ -86,18 +86,19 @@ py==1.6.0 \
     --hash=sha256:06a30435d058473046be836d3fc4f27167fd84c45b99704f2fb5509ef61f9af1
 pycodestyle==2.3.1  \
     --hash=sha512:276bc21b4e6898e379be88f3582135a21314460ad20d4b6c3e12825ac1a72082b2c08bb62099e704c3222f879a1098d0b305132fac1095aaa60ca239b9763a2d \
-    --hash=sha512:e58e10d1514edacfa5a844935a0c4643614dc8b656c9b900d8a56928659cc06c30b2b25ad231261d90837bd53495f8d7ba9748b5480902eadd83124cd262e0ac
+    --hash=sha512:e58e10d1514edacfa5a844935a0c4643614dc8b656c9b900d8a56928659cc06c30b2b25ad231261d90837bd53495f8d7ba9748b5480902eadd83124cd262e0ac \
+    # pyup: <2.4.0
 pydocstyle==2.1.1 \
     --hash=sha512:7bf9e7f35af02cf2da7090316e4c42bd21015604ef6388e88feae7322843b407097bbbb7d12eadbdf179f94db2f7502a8eafc17464e27e5262ccb310be5ef2cf \
     --hash=sha512:815c604c81e6cf7824dd29af24797534e078860d4bbbaa35962beebc7284fe27e2c4238fa806a57225680f0d96c2bef544e0243a98efdc61ac59253aba7d5e5c \
-    --hash=sha512:e3cc922df3fa2c581d9086dbfc7fc2618dc4b2ea5d9b41b7b15625c02a1110e6390b771f696086b32f2c0635f2c0d4ced779211dcaf960d9660e0f52d9269c1d \
-    # pyup: <2.4.0,>=2.0.0
+    --hash=sha512:e3cc922df3fa2c581d9086dbfc7fc2618dc4b2ea5d9b41b7b15625c02a1110e6390b771f696086b32f2c0635f2c0d4ced779211dcaf960d9660e0f52d9269c1d 
```

Comment:
@Akhliskun I just noticed `# pyup: <2.4.0,>=2.0.0` went away. Are we okay to bump `pydocstyle` to 2.4.0+ next time?

Reply:
@JohanLorenzo That was a mistake :( 
I wanted the pin on the pycodestyle.
If you look at it, its on pydocstyle. 
So I noticed it and fixed it with this PR.